AVOutput/AVOutput.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+* If not stated otherwise in this file or this component's LICENSE file the
+* following copyright and licenses apply:
+*
+* Copyright 2022 Sky UK
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <string>
+#include "AVOutput.h"
+#include "UtilsIarm.h"
+
+namespace WPEFramework {
+namespace Plugin {
+
+    SERVICE_REGISTRATION(AVOutput,1, 0);
+
+    AVOutput::AVOutput()
+    {
+        LOGINFO("Entry\n");
+
+        //Common API Registration
+        LOGINFO("Exit \n");
+    }
+
+    AVOutput::~AVOutput()
+    {
+        LOGINFO();
+    }
+
+    const std::string AVOutput::Initialize(PluginHost::IShell* service)
+    {
+	LOGINFO("Entry\n");
```

Comment:
can we move entry exit logs to debug level

Reply:
We need that log to capture the API entry and Exit point for debug purpose.